automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/notebooks/NotebooksSpec.scala

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.dsde.firecloud.test.notebooks
+
+import java.util.UUID
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.fixture.UserFixtures
+import org.broadinstitute.dsde.firecloud.page.workspaces.notebooks.WorkspaceNotebooksPage
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.{Config, UserPool}
+import org.broadinstitute.dsde.workbench.fixture.{MethodFixtures, WorkspaceFixtures}
+import org.broadinstitute.dsde.workbench.service.test.{CleanUp, WebBrowserSpec}
+import org.openqa.selenium.WebDriver
+import org.scalatest.{FreeSpec, Matchers}
+
+import scala.util.Try
+
+class NotebooksSpec extends FreeSpec with WebBrowserSpec with CleanUp with WorkspaceFixtures with UserFixtures with MethodFixtures with Matchers with LazyLogging {
+
+  val billingProject: String = Config.Projects.default
+  val numberOfWorkersError = "Error: Google Dataproc does not support clusters with 1 non-preemptible worker. Must be 0, 2 or more."
+  val noNameClusterError = "Cluster name cannot be empty"
+
+  "User should be able to create and delete a cluster" in withWebDriver { implicit driver =>
+    val user = UserPool.chooseStudent
+    implicit val authToken: AuthToken = user.makeAuthToken()
+    withWorkspace(billingProject, "NotebooksSpec_create_delete_cluster") { workspaceName =>
+      withSignIn(user) { _ =>
+        //dataproc names must be lower case
+        val clusterName = "notebooksspec-" + UUID.randomUUID()
+        val notebooksPage = new WorkspaceNotebooksPage(billingProject, workspaceName).open
+
+        // Cluster creation
+        logger.info("Attempting to create dataproc cluster")
+        val createModal = notebooksPage.openCreateClusterModal
+        createModal.createCluster(clusterName)
+        createModal.awaitDismissed
```

Comment:
If you have createCluster call `submit` at the end instead of `clickOk` it will automatically wait for dismissal.

Reply:
Yeah, that's the way I had it before but in my other test, I don't want it to wait for dismissal, when I click OK, I want to see it display an error

Reply:
Ah, I see. Yeah, we don't have a good pattern for dealing with buttons for which clicking might succeed or fail. Still need work on that.